### PR TITLE
adding new qc terms in resp. to #111

### PR DESCRIPTION
**Pull request by mwalzer**
Please see the term description in #111

### CODE REVIEW

File: psi-ms.obo [hunk 1]
```diff
@@ -22682,6 +22682,35 @@ relationship: has_optional_column: MS:1000767 ! native spectrum identifier forma
 relationship: has_column: UO:0000191 ! fraction
 relationship: has_relation MS:1000285 ! total ion current
 
+[Term]
+id: MS:4000069
+name: m/z acquisition range
+def: "Upper and lower limit of m/z precursor values at which MSn spectra are recorded." [PSI:QC]
+is_a: MS:4000004 ! n-tuple
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_metric_category MS:4000012 ! single run based metric
```
Comment (bittremieux) [inline psi-ms.obo:22691]: Additional category "MS metric"?
```
relationship: has_metric_category MS:4000019 ! MS metric
```

File: psi-ms.obo [hunk 2]
```diff
@@ -22682,6 +22682,35 @@ relationship: has_optional_column: MS:1000767 ! native spectrum identifier forma
...
+relationship: has_units MS:1000040 ! m/z
+
+[Term]
+id: MS:4000070
+name: retention time acquisition range
+def: "Upper and lower limit of time at which spectra are recorded." [PSI:QC]
+is_a: MS:4000004 ! n-tuple
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_metric_category MS:4000012 ! single run based metric
```
Comment (bittremieux) [inline psi-ms.obo:22700]: Additional category "retention time metric"?
```
relationship: has_metric_category MS:4000016 ! retention time metric
```

File: psi-ms.obo [hunk 3]
```diff
@@ -22682,6 +22682,35 @@ relationship: has_optional_column: MS:1000767 ! native spectrum identifier forma
 relationship: has_column: UO:0000191 ! fraction
 relationship: has_relation MS:1000285 ! total ion current
 
+[Term]
+id: MS:4000069
+name: m/z acquisition range
+def: "Upper and lower limit of m/z precursor values at which MSn spectra are recorded." [PSI:QC]
```
Comment (bittremieux) [inline psi-ms.obo:22688]: This seems to refer to traditional DDA MS/MS spectra? For DIA or BoxCar acquisition modes, the m/z range is relevant for MS1 scans as well, and will take different values. I don't think this term needs to change, I just wanted to briefly highlight these related use cases.

File: psi-ms.obo [hunk 4]
```diff
@@ -22682,6 +22682,35 @@ relationship: has_optional_column: MS:1000767 ! native spectrum identifier forma
...
+def: "Upper and lower limit of time at which spectra are recorded." [PSI:QC]
+is_a: MS:4000004 ! n-tuple
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_metric_category MS:4000012 ! single run based metric
+relationship: has_units UO:0000010 ! second
+
+[Term]
+id: MS:4000071
+name: number of chromatograms
+def: "The number of chromatograms recorded for the run." [PSI:QC]
```
Comment (bittremieux) [inline psi-ms.obo:22706]: I don't fully understand what this means. Chromatograms are not directly measured, mass spectra are. Chromatograms are derived by integrating data across the retention time, but there are many ways to do this.

File: psi-ms.obo [hunk 5]
```diff
@@ -22682,6 +22682,35 @@ relationship: has_optional_column: MS:1000767 ! native spectrum identifier forma
...
 relationship: has_relation MS:1000285 ! total ion current
 
+[Term]
+id: MS:4000069
+name: m/z acquisition range
+def: "Upper and lower limit of m/z precursor values at which MSn spectra are recorded." [PSI:QC]
+is_a: MS:4000004 ! n-tuple
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_metric_category MS:4000012 ! single run based metric
+relationship: has_units MS:1000040 ! m/z
```
Comment (bittremieux) [inline psi-ms.obo:22692]: Specify that it is a `range` value concept?
```
relationship: has_value_concept STATO:0000035 ! range
```

File: psi-ms.obo [hunk 6]
```diff
@@ -22682,6 +22682,35 @@ relationship: has_optional_column: MS:1000767 ! native spectrum identifier forma
...
+
+[Term]
+id: MS:4000070
+name: retention time acquisition range
+def: "Upper and lower limit of time at which spectra are recorded." [PSI:QC]
+is_a: MS:4000004 ! n-tuple
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_metric_category MS:4000012 ! single run based metric
+relationship: has_units UO:0000010 ! second
```
Comment (bittremieux) [inline psi-ms.obo:22701]: Specify that it is a `range` value concept?
```
relationship: has_value_concept STATO:0000035 ! range
```

File: psi-ms.obo [hunk 7]
```diff
@@ -22682,6 +22682,35 @@ relationship: has_optional_column: MS:1000767 ! native spectrum identifier forma
...
+def: "Upper and lower limit of m/z precursor values at which MSn spectra are recorded." [PSI:QC]
+is_a: MS:4000004 ! n-tuple
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_metric_category MS:4000012 ! single run based metric
+relationship: has_units MS:1000040 ! m/z
+
+[Term]
+id: MS:4000070
+name: retention time acquisition range
+def: "Upper and lower limit of time at which spectra are recorded." [PSI:QC]
```
Comment (bittremieux) [inline psi-ms.obo:22697]: I'd again explicitly specify "retention time" here to clearly disambiguate from wall time (or other times).

File: psi-ms.obo (hunk 3 above)
Comment (mwalzer) [reply psi-ms.obo:22688]: Should we add a comment there?

File: psi-ms.obo (hunk 4 above)
Comment (mobiusklein) [reply psi-ms.obo:22710]: There are acquisition modes like selected ion monitoring (SIM) and selected reaction monitoring (SRM) where the raw data is stored as chromatograms and we have to ask for them to be sliced into spectra during reading/conversion.

I've extracted UV chromatograms from Thermo's .raw files too. Kind of inconvenient to do but it is built in.

Comment (bittremieux) [reply psi-ms.obo:22710]: Ok, makes sense. Thanks both for clarifying.

**Review by bittremieux — APPROVED** [review]
Looks good to me with the changes.

One more thing that needs to change is to use `PSI:MS` for the dbxrefs, similar to the other PRs.

**Comment by mobiusklein** [comment]
If this is all set, we can merge it.

**Comment by bittremieux** [comment]
Yes, I think it can be merged.

**Review by mobiusklein — APPROVED** [review]

**Pull request merged**